releasenotes/notes/decompose-status-further-with-subcommands-8fd7f1282e990449.yaml

```diff
@@ -0,0 +1,10 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - Add `agent status component [name]` subcommand to fetch only given section status (e.g. `datadog-agent status component forwarder` to get forwarder status).
```

Comment:
```suggestion
  - Add `agent status component [name]` subcommand to fetch only given section status (for example, `datadog-agent status component forwarder` to get forwarder status).
```

Reply:
👍 done in 9eb6fe742075a9bded37a70aa8d27bc76b5b7659